### PR TITLE
Fix bug in _from_hgvs when using SeqRepoDataProxy

### PR DESCRIPTION
**Pull request by korikuzma**
`_from_hgvs` works when using SeqRepoRestDataProxy, but failed when using SeqRepoDataProxy for identity and dup HGVS expressions. This failed because `fetch_uri` expects a namespace and accession (ex: refseq:NM_12345) and `sv.ac` would only provide an accession (ex: NM_12345), so switched to using `sequence_id` as the identifier. 

> def fetch_uri(self, uri, start=None, end=None):
        """fetch sequence for URI/CURIE of the form namespace:alias, such as
        NCBI:NM_000059.3.
>   
>        """
>    
>       namespace, alias = uri_re.match(uri).groups()
>       AttributeError: 'NoneType' object has no attribute 'groups'

Added tests for both SeqRepoRestDataProxy and SeqRepoDataProxy.

### CODE REVIEW

**Comment by larrybabb** [comment]
Deferring to Alex and Reece while on vacation.

**Review by larrybabb — APPROVED** [review]

**Comment by reece** [comment]
I'll look later today.

File: tests/extras/test_translator.py [hunk 1]
```diff
@@ -17,23 +17,31 @@
 }
 
 
+@pytest.mark.parametrize("translator", ("rest_tlr","tlr"))
```
Comment (reece) [inline tests/extras/test_translator.py:20]: Well, that's new to me! Nice!

Minor comment: `translator` is really a translator selector or name, not a Translator instance. Please consider a different name to make that clearer. `tlr_fixture_name`?

**Review by reece — COMMENTED** [review]
Nice job, @korikuzma!
There's one minor suggestion. Please comment and we can merge shortly.

**Comment by korikuzma** [comment]
@reece Applied your suggestion!

**Comment by reece** [comment]
Hi @korikuzma : This all *looks* good, but it's failing tests for me locally. And, due to travis-ci.org shutdown, we don't have CI/CD testing right now. :-( 

I will need to resolve that before merging. Please stand by a couple of days.

**Comment by reece** [comment]
Okay, hours later... I think I just got testing working.  Please merge main into your branch and resolve conflicts.

**Comment by reece** [comment]
I merged this before it was ready. #60 reverts this merge. @korikuzma : You'll need to create another PR. Sorry.

**Pull request merged**